src/bramble/client/main.js

```diff
@@ -147,6 +148,7 @@ define([
             return;
         }
 
+        ProjectStats.init(root,function(){});
```

Comment:
You're going to have a race condition here, since you are calling `init` and it will take time to walk the root dir, but then you immediately tell the instance to `mount` below.  You are also silently ignoring the error case.  You need to have them happen in succession.  Do it like this:

```js
ProjectsStats.init(root, function(err) {
  if(err) {
    // Deal with error case here, probably something like:
    setReadyState(Bramble.ERROR, new Error("Unable to access filesystem: ", err));
    return;
  }
  _instance.mount(root, filename);
});
```